inc/minishell.h

```diff
@@ -41,32 +40,32 @@ typedef struct s_token
 	char			*value;
 }					t_token;
 
-extern volatile sig_atomic_t	g_sigint_received;
+//signal_handler.c
+void	set_handler_one(struct sigaction *sa);
+void	set_handler_two(struct sigaction *sa);
 
-typedef struct s_program
```

Comment:
what? Where it came from? I think i didnt see it before or im too tired

Reply:
I copied everything from my pipex and didn't delete until now